protocol/0062-SPAM-spam_protection.md

```diff
@@ -83,6 +83,17 @@ Further, each party is allowed to submit up to `n` transactions per epoch where
 
 **Note** `spam.protection.max.updatePartyProfile` must be an integer greater than or equal to `0` (and default to `5`).
 
+### Transaction Spam
+
+Network parameter: `spam.order.MinimalMarginQuantumMultiple` (between 0 and infinite)
+
+If the maintenance margin for a given transaction is smaller than the parameter `spam.order.MinimalMarginSizeQuantumMultiple`, then the transaction is pre-block rejected.
```

Comment:
Is the minimal margin size the right thing to do here? I kinda remember us discussing instead using the quantum notional value of the order as it'd be much less computing heavy?

Reply:
Margin is what we want because it takes into account leverage. 

In terms of computation heavyness:  take `(rf + linear scaling param) x size x price` it's not much heavier than just `size x price` no?